pkg/generator/ignition/ignition.go

```diff
@@ -32,41 +30,28 @@ ExecStart=/usr/bin/containerd --config=/etc/containerd/config.toml
 // Starting with ignition 2.0, ignition itself contains the required parsing logic, so we can use ignition directly.
 // see https://github.com/coreos/ignition/blob/master/config/config.go#L38
 // Therefore we must update ignition to 2.0.0 in the images and transform the gardener config to the ignition config types instead.
-func IgnitionFromOperatingSystemConfig(ctx context.Context, c client.Client, config *extensionsv1alpha1.OperatingSystemConfig) ([]byte, error) {
+func IgnitionFromOperatingSystemConfig(config *generator.OperatingSystemConfig) ([]byte, error) {
```

Comment:
We can move this into generator.go an make it private, ignition_test.go can then be renamed to genertor_test.go. The whole package is named ignition this make all obvious anyway.

Reply:
Done.